bchrpc/server.go

```diff
@@ -1688,6 +1718,24 @@ func marshalTransaction(tx *bchutil.Tx, confirmations int32, blockHeader *wire.B
 	return respTx
 }
 
+// setInputMetadata will set the value, previous script, and address for each input in the mempool transaction
+// from blockchain data adjusted upon the contents of the transaction pool.
+// Used when no s.txIndex is available
+func setInputMetadata(respTx *pb.Transaction, txDesc *rpcEventTxAccepted, view *blockchain.UtxoViewpoint, chainParams *chaincfg.Params) {
```

Comment:
Only change I would request is to change the name of this function as it has the same name as a server method. Maybe something like `setInputMetadataFromView`.

Reply:
Done